applications/nrf5340_audio/src/audio/audio_datapath.c

```diff
@@ -747,6 +756,34 @@ static void audio_datapath_i2s_stop(void)
 	alt_buffer_free_both();
 }
 
+static bool update_cmpr_val_check_on(uint32_t seq_num)
+{
+	/*  Checks if seq_num has wrapped, if so updates offset */
+	if (seq_num < sync_led_last_seq_num) {
+		if (seq_num >= (sync_led_last_seq_num -
+				(SYNC_LED_SEQ_NR_WRAP_LIMIT - (SYNC_LED_SEQ_NR_PERIOD / 2)))) {
+			sync_led_last_seq_num = seq_num;
+			sync_led_seq_num_offset = seq_num;
+			return true;
+		}
+	} else {
+		if (seq_num >= sync_led_last_seq_num + (SYNC_LED_SEQ_NR_PERIOD / 2)) {
+			sync_led_last_seq_num += (SYNC_LED_SEQ_NR_PERIOD / 2);
+			return true;
+		}
+	}
+	return false;
+}
+
+static bool sync_led_toggle_check(uint32_t seq_num, uint32_t offset)
+{
+	if ((seq_num % SYNC_LED_SEQ_NR_PERIOD) - offset == SYNC_LED_SEQ_NR_ON) {
```

Comment:
It seems that we can run into a missed seq_num issue here as well. If this if is meant to return true at e.g. `seq_num` = 3000, but if we miss 3000 and get 3001 instead, it should still turn on?

Reply:
Added new suggestion